Documentation/network/servicemesh/ingress.rst

```diff
@@ -43,14 +43,15 @@ path prefix) between resources.
 This is a step-by-step guide on how to enable the Ingress Controller in
 an existing K8s cluster with Cilium installed.
 
+.. _Kubernetes Ingress: https://kubernetes.io/docs/concepts/services-networking/ingress/
+
 Prerequisites
 #############
 
-* Cilium must be configured with ``kubeProxyReplacement`` as partial
-  or strict. Please refer to :ref:`kube-proxy replacement <kubeproxy-free>`
-  for more details.
-* Cilium must be configured with the L7 proxy enabled using the ``--enable-l7-proxy`` flag (enabled by default).
-* The minimum supported Kubernetes version for Ingress is 1.19.
```

Comment:
I think we still need this one, before 1.19, Ingress resource is with a different API version (i.e. beta)

Reply:
But Cilium with K8s < 1.19 is no longer supported, is it?

Reply:
From https://docs.cilium.io/en/v1.14/network/kubernetes/requirements/#kubernetes-version